src/config.h

```diff
@@ -67,3 +67,5 @@ bool Config_Option_set(Config_Option_Field field, RedisModuleString *val);
 
 bool Config_Option_get(Config_Option_Field field, ...);
 
+// Sets the callback function which should be called when a config param changes.
+void Config_Subscribe_Changes(void (*callback)(Config_Option_Field type, void *val));
```

Comment:
`void Config_Subscribe_Changes(function_type cb);`
set config.cb to input: `config.cb = cb;`

all files should end with newline, usually the last function in a file is the Free function.